manager-bundle/skeleton/config/security.yaml

```
@@ -19,6 +19,10 @@ security:
             contao_login:
                 remember_me: false
 
+            login_throttling:
+                max_attempts: 5
+                interval: 1 minute
```

```suggestion
            login_throttling: ~
```

According to the docs this should use the default of 5 per minute. ☺️

---

Correct, I thought to make it more obvious where to change what.